### PR TITLE
Visually hide section label on desktop

### LINKED ISSUE #1791: Visually hide section label on desktop.
**Is your feature request related to a problem? Please describe.**
Currently the `SectionLabel` is visually hidden for widths under `600px`. While viewing on desktop widths, `SectionLabel` will be visible.

**Describe the solution you'd like**
Update `SectionLabel` such that it is visually hidden under all breakpoints.

Remember that this should be merged together with the desktop top stories into Simorgh to avoid losing the section label without desktop width top stories.

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Visual Checks.

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by OlgaLyubin**
Resolves #1791 

**Overall change:** 
Updated a SectionLabel component so that it is visually hidden under all breakpoints when visuallyHidden prop is true (for top stories).

**Code changes:**
Changed a media query so that a SectionLabel component would be visually hidden for top stories.
Updated test snapshots to match.

**Important:**
This PR should only be merged together with the desktop top stories.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

**Comment by dr3** [comment]
Chatted IRL, I think the issue for this might be wrong, and that we should never show the section label. This PR does implement the issue correctly, showing it after 1280, but that seems weird as a UX decision :P

@OlgaLyubin to confirm :)

**Review by andrew-nowak — COMMENTED** [review]
looking good - version needs incrementing in package.json, package-lock.json, and CHANGELOG.md in psammead-section-label, then lgtm

**Comment by DenisHdz** [comment]
> looking good - version needs incrementing in package.json, package-lock.json, and CHANGELOG.md in psammead-section-label, then lgtm

You will also have to update the README, removing the references to `600px` breakpoint.

**Review by andrew-nowak — APPROVED** [review]
lgtm

**Review by DenisHdz — APPROVED** [review]
LGTM

**Review by ghost — COMMENTED** [review]
Looks great. Could we be consistent with the use of 
`under all breakpoints`, `for all breakpoints`, `at all breakpoints`

File: packages/components/psammead-section-label/src/index.jsx [hunk 1]
```diff
@@ -44,15 +41,13 @@ const SectionLabelWrapper = styled.div`
...
-      // Hide when under 600px
-      @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-        clip-path: inset(100%);
-        clip: rect(1px, 1px, 1px, 1px);
-        height: 1px;
-        overflow: hidden;
-        position: absolute;
-        width: 1px;
-      }
+      // Hide for all breakpoints for top story
```
Comment (dr3) [inline packages/components/psammead-section-label/src/index.jsx]:
```suggestion
      // Hide for all breakpoints
```
Doesnt need to be top story, just `visuallyHidden` is set :)

**Review by dr3 — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**